samples/WorkerBindingSamples/Blob/BlobTriggerBindingSamples.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace SampleApp
+{
+    public static class BlobTriggerBindingSamples
+    {
+        [Function(nameof(BlobClientFunction))]
+        public static async Task BlobClientFunction(
+            [BlobTrigger("client-trigger/{name}", Connection = "AzureWebJobsStorage")] BlobClient client,
+            FunctionContext context)
+        {
+            var logger = context.GetLogger(nameof(BlobClientFunction));
```

Comment:
Any reason why you're using FunctionContext in these samples, rather than taking the logger as a ctor param as you did in the input samples?

Reply:
Wanted to show that both options are available, but I think we want folks to use the ctor pattern right? Happy to switch these to match input bindings